code/game/jobs/job/civilians/support/synthetic.dm

```diff
@@ -4,12 +4,12 @@
 	spawn_positions = 1
 	allow_additional = 1
 	scaled = 1
-	supervisors = "the acting commanding officer"
+	supervisors = "the acting commander, command staff"
 	selection_class = "job_synth"
 	flags_startup_parameters = ROLE_ADD_TO_DEFAULT|ROLE_ADMIN_NOTIFY|ROLE_WHITELISTED|ROLE_CUSTOM_SPAWN
 	flags_whitelist = WHITELIST_SYNTHETIC
 	gear_preset = /datum/equipment_preset/synth/uscm
-	entry_message_body = "You are a <a href='"+WIKI_PLACEHOLDER+"'>Synthetic!</a> You are held to a higher standard and are required to obey not only the Server Rules but Marine Law and Synthetic Rules. Failure to do so may result in your White-list Removal. Your primary job is to support and assist all USCM Departments and Personnel on-board. In addition, being a Synthetic gives you knowledge in every field and specialization possible on-board the ship. As a Synthetic you answer to the acting commanding officer. Special circumstances may change this!"
+	entry_message_body = "You are a <a href='"+WIKI_PLACEHOLDER+"'>Synthetic!</a> You are held to a higher standard and are required to obey not only the Server Rules but Marine Law and Synthetic Rules. Failure to do so may result in your White-list Removal. Your primary job is to support and assist all USCM Departments and Personnel on-board. In addition, being a Synthetic gives you knowledge in every field and specialization possible on-board the ship. As a Synthetic you answer to the acting commander and command staff. Special circumstances may change this!"
```

Comment:
I'm not sure this is exactly correct or it seems awkward this way. I'd just leave it as the acting commander since that's going to be the value for most roles anyways.

Reply:
Synth guidelines require players to follow command staff's orders.
"Do not disobey valid orders from Command personnel. An unlawful order is not valid, thus can be ignored."

I'll leave the final call up to you, I only had it this way because that specifically requires synthetics to follow it. I thought it would be a good reminder for newer synthetics.

Reply:
When I asked the council they said it was that you need to listen to 2nd Lt and above not that you answer to them directly. There were also concerns that people could abuse this wording. So no I still think it should be how it was.